docs/snippets/package/aws-with-vpc/crossplane.yaml

```diff
@@ -6,3 +6,9 @@ metadata:
     guide: quickstart
     provider: aws
     vpc: new
+spec:
+  crossplane:
+    version: ">=v0.14.0-0"
+  dependsOn:
+    - provider: crossplane/provider-aws
+      version: ">=v0.14.0-0"
```

Comment:
should these AWS ones be 0.15?

Reply:
They could be, but I don't these packages explicitly depend on anything in v0.15.0, but they will use the latest version if aws isn't already installed. So keeping the constraint here makes us compatible with older versions, while defaulting to the newest.